### DISCUSSION TITLE
One GPU not showing up in a 5 GPU rig

### DISCUSSION THREAD

**Discussion by Kaszebe** [LostRuins/koboldcpp#1071] [Q&A]
I have four 3090s and one 4090 on an RomeD8-2T/EPYC 7F52 rig with 32GB RAM. I'm running PopOS.

I just installed Kobold last night, and when I run the program, it's only showing 4 GPUs when I click the GPU ID drop-down menu: Three 3090s and the one 4090. 

I ran nvidia-smi, and all five GPUs are showing up. 

I'm using the GUI and not the CLI. 

Anyone know why this could be happening?

Many thanks.

**Comment by LostRuins** [comment]
By default KCPP only displays the first 4 GPUs individually. However, if you select 'All' then it will use al 5, and you can control it with tensor_split

**Comment by Kaszebe** [comment]
Thanks! And with regards to tensor_split...is it in percentages? Or VRAM? I asked ChatGPT and it said something like 2,3,3,3,3.


Here's what it originally said: 

```
For example, if you input 1,1,1,2, it would mean:

Each of the first three GPUs (in this case, the 3090s) gets 1/(1+1+1+2) = 20% of the work
The last GPU (the 4090) gets 2/(1+1+1+2) = 40% of the work
```

**Reply by LostRuins** [reply]
It's a ratio. 4 6 means 40% to 60%
